Gems/Atom/RPI/Code/Source/RPI.Reflect/Image/StreamingImageAsset.cpp

```diff
@@ -172,16 +172,30 @@ namespace AZ
 
         AZStd::span<const uint8_t> StreamingImageAsset::GetSubImageData(uint32_t mip, uint32_t slice)
         {
+            const bool forceLoad = true;
```

Comment:
should this be a cvar or AZ trait? if we don't force load will we just see the "not loaded yet" texture until it gets streamed in?

Reply:
It would be crash for decal feature processor because it's always expecting the data exist.

Reply:
What is the point of forceLoad if it is always true?